tests/PersonnummerTest.php

```diff
@@ -119,6 +122,59 @@ public function testVgrReserveNumbersForMales()
         });
     }
 
+    public function testSllReserveNumbersForMales()
+    {
+        $female = new Personnummer('992004920019');
+        $this->assertEquals('992004920019', $female->format());
+        $this->assertTrue($female->isSllReserveNumber());
+
+        // Wrong check digit:
+        $this->assertThrows(PersonnummerException::class, function () {
+            new Personnummer('992004920018');
+        });
+
+        // Wrong gender:
+        $this->assertFalse($female->isFemale());
```

Comment:
Shouldn't this assert true if this testing number is in fact a female?

Reply:
Also, add test for the reverse method (assert true ->isMale())

Reply:
✅